### DISCUSSION TITLE
Ignore devices

### DISCUSSION THREAD

**Discussion by ppernet** [samemory/homebridge-eufy-security-control#297] [Q&A] [answered]
Hello, question about optionals settings in Config JSON : how to stop some EUFY devices to be exposed to homekit ? 
For exemple, I don't want see keypad in home app. I suppose it works with ignoreDevices entry but where can i find names to enter between the [] ? 

Thank you for your help.

**Comment by lenoxys** [comment, marked as answer]
Restart the wizard setup the list will be displayed and you can check/uncheck to ignore.
